README.md

```diff
@@ -1,4 +1,4 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/v7w84n21w9yjspq7/branch/master?svg=true)](https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/v7w84n21w9yjspq7/branch/main?svg=true)](https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/main)
```

Comment:
This URL doesn't seem to exist, it can't find build.

Reply:
That's strange, it should have been created by the prs appveyor run

Reply:
Hmmm, from what I can tell it should be created when the PR is merged in and the main becomes the current build

Reply:
Cool well were about to find out. Merging.